.github/workflows/ci.yml

```diff
@@ -33,4 +39,5 @@ jobs:
           python -m pip install --upgrade coverage[toml] virtualenv tox tox-gh-actions          
 
       - name: "Run tox targets for ${{ matrix.python-version }}"
+        shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
```

Comment:
Do you mind leaving a comment as to what this does / why it's necessary?

I can imagine us looking at it in a year and having no idea :sweat_smile:

Reply:
Good thinking!